src/main/java/org/apache/mesos/specification/TaskTypeSpecification.java

```diff
@@ -0,0 +1,8 @@
+package org.apache.mesos.specification;
+
+/**
+ * A TaskTypeSpecification describes the count of a particular TaskSpecification which should be launched.
```

Comment:
- Why does the TypeSpecification extension just add a count? This name feels odd
- Any reason to not just merge this into TaskSpecification? What is the count of?